### ISSUE TITLE
DAG is in running state but tasks are not getting triggered

### ISSUE THREAD

**Issue by shash95** [puckel/docker-airflow#254]
I followed the instructions in the README file to get Airflow up and running inside the container, and found that although the DAG gets into running state (on the UI), the tasks within the DAG seem to be waiting indefinitely and never actually get triggered.
Given that some of the steps given in the documentation are optional, I followed the following steps in order to at least get the example DAGs up and running within my container - 

_Pulled the image from dockerhub_

`docker pull puckel/docker-airflow`

_Triggered Airflow with default settings, which should start it with Sequential Executor_

`docker run -d -p 8080:8080 -e LOAD_EX=y puckel/docker-airflow`

I'm relatively new to _setting up_ Airflow and dockers, although I have worked on Airflow in the past. So, it's possible that I am missing something very basic here, since no one else seems to be facing the same issue. Any help would be highly appreciated.

**Comment by alexterner** [comment]
Same problem

**Comment by alexterner** [comment]
Run this command: docker-compose -f docker-compose-LocalExecutor.yml up -d
worked for me

**Comment by johnlovesdata** [comment]
If you're running the SequentialExecutor, you'll have to spin up the scheduler separately. You can do that by opening a terminal once the docker is up and running
`docker exec -it <CONTAINER ID> bash`
And then 
`airflow scheduler`

Looking into the code in entrypoint.sh, the scheduler only runs when spinning up the container using an executor that isn't sequential, which is why @alexterner command also will work.

**Comment by venuktan** [comment]
same problem, any fixes

**Comment by zhongjiajie** [comment]
Hi everyone, I review the `entrypoint.sh` code and fix it in https://github.com/puckel/docker-airflow/pull/314. If maintainer do not merge my PR. You can do some step like below

* Clone repo: `git clone git@github.com:puckel/docker-airflow.git`
* Change code in `script/entrypoint.sh` like [my PR](https://github.com/puckel/docker-airflow/pull/314/files#diff-fc400973c8921a5d2c726bad83481b71R73)
* Create new dockerfile named `Dockerfile1` in repo
* Add content like below and save it
```Dockerfile
FROM puckel/docker-airflow

COPY script/entrypoint.sh /entrypoint.sh

ENTRYPOINT ["/entrypoint.sh"]
CMD ["webserver"]
```
* Build docker image by yourself: `docker build --no-cache -f Dockerfile1 -t puckel/docker-airflow:test .`
* Run command: `docker run -d -p 8080:8080 -e LOAD_EX=y puckel/docker-airflow:test`

It will fix the bug.

**Comment by nasmithan** [comment]
I'm having this issue with celery executor. Have about 75 "running" dags, with only 4 running tasks. It gets in this bad state every week or so.

**Comment by carbolymer** [comment]
In my case, a docker container with the worker was restarting without the end. Turns out in airflow 1.10.3 when you run `aiflow worker` it forks to background by default and exits, even when you don't pass `-d` parameter. In my case I had to change entrypoint to:
```
/entrypoint.sh worker && tail -f /dev/null
```

**Comment by mikolaje** [comment]
- check if airflow scheduler is running
- check if airflow webserver is running
- check if all DAGs are set to On in the web UI
- check if the DAGs have a start date which is in the past
- check if the DAGs have a proper schedule (before the schedule date) which is shown in the web UI
- check if the dag has the proper `pool` and `queue`.

**Comment by eightlimbed** [comment]
Thanks for sharing that checklist @mikolaje -- For me, I was using the CeleryExecutor and I didn't define `queue`, so my DAG would run, but all the tasks within the DAG stayed in a scheduled state because (I assume) the scheduler didn't know where to send them to be executed.

**Comment by lestatkim** [comment]
@mikolaje thanks 🤘🏼

**Comment by puckel** [comment]
Hi, fixed in #359, released in 1.10.6.

**Comment by LeoDoldan7** [comment]
If the problem is a subdag, it may be just paused. But the pause/unpause button doesn't appear in Tree View nor Graph View.

You have to go to the Logs, and you will have the button at the top-right corner.
